config/project/config.go

```diff
@@ -60,4 +60,14 @@ func Configure(p *config.Provider) {
 		r.Kind = "ProjectTemplate"
 		r.ShortGroup = "project"
 	})
+
+	// This is whats not working!
+	p.AddResourceConfigurator("infisical_secret_sync_github", func(r *config.Resource) {
+		r.Kind = "SecretSyncGithub"
+		r.ShortGroup = "secretsync" // lowercase not allowed
+
+		r.References["project_id"] = config.Reference{
+			TerraformName: "infisical_project",
+		}
+	})
```

Comment:
this should be in a different file/directory... the current one is primarily for ShortGroup="project".

Just noticed as well that the secrets resource is misplaced here